flatpak/key-mapper.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+PYTHONPATH="/app/usr/lib/python3.8/site-packages/" python3.8 /app/usr/bin/key-mapper-gtk
```

Comment:
does flatpak deliver all python versions, or is it possible that somebody would have 3.7 instead of 3.8?

Reply:
These are isolated containers. If there's python3.8, as I understand it should be always there. It's what is provided as a standard Gnome environment https://docs.flatpak.org/en/latest/available-runtimes.html.

So these are not host files, by files inside virtual machine / container.